component-sets/default-components/components-definition.json

```diff
@@ -179,6 +198,15 @@
             "name": "text",
             "components": ["intro", "body", "footer", "quote", "separator", "crosshead", "author"]
         },
+        {
+            "name": "data",
+            "label": { "key": "GROUP_DATA_LABEL" },
+            "logo": {
+                "icon": "icons/group-logos/data.svg",
```

Comment:
Why `group-logos` and not `groups`? Name-wise this ties it completely to logos.

Reply:
It's a good idea indeed to name it something more generic to not restrict what the icons are for. Hadn't kept that in mind.